docs/.vuepress/utils/KDatePickerMockData.js

```diff
@@ -0,0 +1,189 @@
+const CHART_DATA_REQUEST_TRIM = 10
```

Comment:
nit: can you add a comment at the top of this file that explains this is for the docs site examples only?